src/Sylius/Behat/Context/Api/Admin/ManagingCurrenciesContext.php

```diff
@@ -107,6 +109,32 @@ public function iShouldBeNotifiedThatCurrencyCodeMustBeUnique(): void
         Assert::same($this->responseChecker->getError($response), 'code: Currency code must be unique.');
     }
 
+    /**
+     * @Then I should be notified that a code is required
+     */
+    public function iShouldBeNotifiedThatACodeIsRequired(): void
+    {
+        $response = $this->client->getLastResponse();
+        Assert::false(
+            $this->responseChecker->isCreationSuccessful($response),
+            'Currency has been created successfully, but it should not',
+        );
+        Assert::same($this->responseChecker->getError($response), 'code: Please choose currency code.');
+    }
+
+    /**
+     * @Then I should be notified that the code is invalid
+     */
+    public function iShouldBeNotifiedThatTheCodeIsInvalid(): void
+    {
+        $response = $this->client->getLastResponse();
+        Assert::false(
+            $this->responseChecker->isCreationSuccessful($response),
+            'Currency has been created successfully, but it should not',
+        );
+        Assert::same($this->responseChecker->getError($response), 'code: This value is not a valid currency code.');
+    }
```

Comment:
These methods seem to have common that can be extracted to a separate method with a message as argument